src/main/scala/monocle/Optic.scala

```diff
@@ -4,202 +4,184 @@ import monocle.impl._
 import monocle.functions.*
 import monocle.internal.*
 
-type Optic[+ThisCan, S, A]      = POptic[ThisCan, S, S, A, A]
-type OpticGet[+ThisCan, -S, +A] = POptic[ThisCan, S, Nothing, A, Any]
+type Optic[+Can, S, A]      = POptic[Can, S, S, A, A]
+type OpticGet[+Can, -S, +A] = POptic[Can, S, Nothing, A, Any]
 
-object GetOptic:
+object OpticGet:
   def id[A]: OpticGet[Get, A, A] =
     Optic.thatCan.get[A, A](identity)
 
 object Optic:
-  def id[A]: Optic[Get & ReverseGet, A, A] =
+  def id[A]: Optic[ConvertBetween, A, A] =
```

Comment:
Let me think about it. I quite liked the intersection type but that might be just me.